config/prometheus/service.yaml

```diff
@@ -6,15 +6,14 @@ metadata:
   namespace: openshift-machine-api
   annotations:
     exclude.release.openshift.io/internal-openshift-hosted: "true"
-    include.release.openshift.io/self-managed-high-availability: "true"
     service.alpha.openshift.io/serving-cert-secret-name: cluster-baremetal-operator-tls
   labels:
     k8s-app: cluster-baremetal-operator
 spec:
   type: ClusterIP
   ports:
   - name: https
-    port: 8443
+    port: 9444
```

Comment:
FWIW, neither CBO nor MAO are using host networking, so there is no port conflict.

There _would_ be a problem if they were both using the same labels so that either of the Services' selectors ended up selecting both Pods, but `k8s-app: cluster-baremetal-operator` should be pretty unique.